### DISCUSSION TITLE
How to get rid of icons next to line numbers.

### DISCUSSION THREAD

**Discussion by evwltrs** [LazyVim/LazyVim#2936] [Q&A]
![image](https://github.com/LazyVim/LazyVim/assets/20270765/b88b3d4d-c572-4c22-9768-df1fb71aff0f)

I'd like to get rid of these icons.

**Comment by dpetka2001** [comment]
```lua
-- ~/.config/nvim/lua/plugins/lsp.lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      diagnostics = {
        signs = false,
      },
    },
  },
  {
    "lewis6991/gitsigns.nvim",
    opts = {
      signcolumn = false,
    },
  },
}
```
The left icons come from LSP diagnostics. The right icon comes from `gitsigns`.

**Reply by zvrkSam** [reply]
I tried only disabling the `nvim-lspconfig` part like you suggested, but I get a error:
```
Failed to run `config` for nvim-lspconfig

...share/nvim/lazy/LazyVim/lua/lazyvim/plugins/lsp/init.lua:123: attempt to index field 'signs' (a boolean value)
```

**Reply by dpetka2001** [reply]
Neovim version? Stable?

**Reply by zvrkSam** [reply]
0.9.5 yes it is the stable one

**Reply by dpetka2001** [reply]
You have to wait for #2897 to get merged or use nightly in the meantime.

**Reply by zvrkSam** [reply]
Thanks for the reply, I will wait in the meantime, nightly has other issues that I don't want to deal with.
